samples/mqtt/basic_pub_sub/main.cpp

```diff
@@ -22,7 +24,9 @@ static void s_printHelp()
         stdout,
         "basic-pub-sub --endpoint <endpoint> --cert <path to cert>"
         " --key <path to key> --topic --ca_file <optional: path to custom ca>"
-        " --use_websocket --signing_region <region> --proxy_host <host> --proxy_port <port>\n\n");
+        " --use_websocket --signing_region <region> --proxy_host <host> --proxy_port <port>"
+        " --x509 --x509rolealias <role_alias> --x509endpoint <endpoint> --x509thing <thing_name>"
+        " --x509cert <path to cert> --x509key <path to key> --x509rootca <path to root ca>\n\n");
```

Comment:
need to update these flags with underscores

Reply:
gdi